packages/core/src/components/Table/renderers/renderers.tsx

```diff
@@ -229,7 +228,8 @@ export function hvDropdownColumn<
           onChange={(val) => onChange?.(row.id, val)}
           disabled={dsbld}
           dropdownProps={{
```

Comment:
I'm not sure about this one because it can lead to breaking changes. It was previously using `setId(id, column.id)` to concat the id provided by the user and the column id. However, it didn't make a lot of sense to be because the user would have to manually set this concat id on the tablet header. 
I can't leave it as it was and only fix the samples. What do you think?

Reply:
If `column.id` automatically correctly labels the element, then fine by me - less work for the user 👍

We could just leave `id` there (unused), that way we wouldn't be _technically_ changing anything in the API